### ISSUE TITLE
"Err: App Not Running" message can be misleading and might cause people to think app is broken when it's not

### ISSUE THREAD

**Issue by JobLeonard** [meshenger-app/meshenger-android#157]
So, first of all, awesome app! My partner and I just installed it on my phone and on an old tablet, and after we got it to work we basically have a high-quality baby monitor with optional camera feed that doesn't need to leave the local network¹.

However, I almost uninstalled it thinking it was broken: it kept refusing to connect, giving the error that the app was not running. This obviously wasn't true because we had both apps side by side. It took us a while to realize what the real problem was: our router uses two separate wifi connections (2.4 gHz and 5Ghz), and the phone and tablet automatically connected to two different ones.

Maybe "Err: Contact Not Found On Current Network" would be a more helpful error message, but I don't know if that might also be misleading in some cases.

¹ btw, "turn old phones and tablets into a baby monitor!" might be a nice selling point to add to the description, no? Anyway, thanks in doing your part in reducing e-waste ;)

**Comment by mwarning** [comment]
Hi there. Nice to hear that it brings values to your life!
The error messages need to be revisted. Maybe I get to do it soon. Thank you for the heads up.

If I may ask, how do you use the App as a baby monitor? Do you have a call open all the time? Do you need to keep the screen on all the time? Is the connection stable over long times?

**Comment by mwarning** [comment]
I have removed that particular error message for now.

**Comment by JobLeonard** [comment]
> If I may ask, how do you use the App as a baby monitor? Do you have a call open all the time? Do you need to keep the screen on all the time? Is the connection stable over long times?

Sure! Basically, if the little one is home during the day then she naps around lunch-time, usually one to two hours, but sometimes a little longer. I work from home but have a separate work office room, and four doors separate the bedroom where she naps from my work room. We want to keep the door to the bedroom closed to reduce chances she wakes up from noises.

So I connect an old tablet to my phone (with the phone connected to my headphones) and set up a one-way audio-stream; the video isn't really needed.

Of course, we changed the app settings to never optimize power usage so the tablet doesn't freeze the app with the screen closed (I don't know if that's really needed or if the app somehow signals to Android that it should remain on during connections, but better to be safe than sorry). Both tablet and phone are connected to chargers during this time anyway, so battery usage isn't a concern, and the tablet screen is turned off.

We used to do this using one of the (many) chat-apps that support audio-calls, but this feels better both from a privacy perspective and an energy use perspective (no energy wasted on roundtrips to data centers)

So far meshenger worked OK the one time we tried it, but she slept less than half an hour that day so it's not like we had to maintain the connection for an extremely long time (well, still quite a long audio call I guess)
